http/base/src/main/java/org/apache/felix/http/base/internal/handler/ServletHandler.java

```diff
@@ -42,6 +44,9 @@ public abstract class ServletHandler implements Comparable<ServletHandler>
 
     private static final String JAVA_SERVLET_TEMP_DIR_PROP = "jakarta.servlet.content.tempdir";
 
```

Comment:
How about moving the extra handling into a subclass and keep ServletHandler generic - and then make a decision when a new ServletHandler class is created whether this or the subclass should be used?

Reply:
Good one, i will look into this later this week probably.

Reply:
Can you take another look @cziegeler?